### PR TITLE
tweak type annotations

### PR DESCRIPTION
**Pull request by JensHeinrich**
## Change Summary

Mark several optional parameters as such

## Related issue number

skip change file check
## Checklist

* [ ] Unit tests for the changes exist
* [x] Tests pass on CI and coverage remains at 100%
* [x] Documentation reflects the changes where applicable
* [x] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)
* [x] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

Selected Reviewer: @samuelcolvin

### CODE REVIEW

**Comment by JensHeinrich** [comment]
please review

File: changes/5499-JensHeinrich.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Mark optional parameters as optional in type annotations.
```
[the remark this answers is not in the document]
Comment (JensHeinrich) [reply changes/5499-JensHeinrich.md]: done

**Review by hramezani — APPROVED** [review]
LGTM 👍

**Comment by hramezani** [comment]
Thanks @JensHeinrich

**Pull request merged**